### ISSUE TITLE
Replace how we send messages.

### ISSUE THREAD

**Issue by Claudi0-V** [waream2/cb-bots#25]
currently some of the messages are sent with `interaction.channel.send` (I can only record of ping using right now). However, due to @emilydta work we found out about `interaction.reply` that will send a message back replying directly to the one sent. by using `interaction.reply` we can remove unnecessary things like pinging the user who sent the message via username, such as "hey @username we gave 1point to @anotherUser".

**Comment by waream2** [comment]
Just to make sure I understand, we're suggesting that we reply directly to the user who called the bot, as opposed to sending a message to the entire server?

**Comment by Claudi0-V** [comment]
nope. `.reply()` directly replies to the message that the bot is responding to. so if I sent `!points`, it would respond to that message in that place (channel and server);

**Comment by waream2** [comment]
Ah okay, I think this makes sense. So it would keep the bots response in scope with the message that prompted it?

**Comment by Claudi0-V** [comment]
yes. if you wanna understand better the difference, try the command "!ping"  vs "!points". ping uses `channel.send()` while points use `.reply()`

**Comment by emilydta** [comment]
I changed  `channel.send()` in ping to `.reply()` so now everything is using that atm.

**Comment by Claudi0-V** [comment]
aight, this issue was open not only for the current ones, but for future reference.i think that we al agree that this way is better in most cases
